modules/remotecontrol/RemoteControl.js

```diff
@@ -25,8 +27,35 @@ class RemoteControl {
      * Constructs new instance. Creates controller and receiver properties.
      */
     constructor() {
+        super();
         this.controller = new Controller();
+        this._active = false;
         this._initialized = false;
+
+        // eslint-disable-next-line no-return-assign
+        this.controller.on('active-changed', active => this.active = active);
```

Comment:
I can't really remember right now. Probably it is related to the `APP.conference.isDesktopSharingEnabled` check which should be relevant only for the receiver. Do you want to change it somehow?